Sources/Views/ALKDocumentCell.swift

```diff
@@ -92,7 +93,7 @@ class ALKDocumentCell: ALKChatBaseCell<ALKMessageViewModel> {
         label.numberOfLines = 1
         label.font = UIFont.systemFont(ofSize: 12)
         label.isOpaque = true
-        label.textColor = .black
+        label.textColor = .white
```

Comment:
hardcoded colors will be same for both modes?

Reply:
As we are not modifying the message sending process, it won't require customization for dark mode support.